static/src/javascripts/projects/commercial/modules/header-bidding/utils.ts

```diff
@@ -166,6 +168,16 @@ export const shouldIncludeImproveDigitalSkin = (): boolean =>
 	(isInUk() || isInRow()) &&
 	getBreakpointKey() === 'D'; // Desktop only
 
+/**
+ * 	Include Criteo if not in the control group of the AB test
+ *
+ *  Equivalent to those in the variant group and those not participating
+ *
+ * TODO check they're not participating in the Smart AB test either!
+ */
+export const shouldIncludeCriteo = (): boolean =>
+	!isInVariantSynchronous(integrateCriteo, 'control');
```

Comment:
When we're ready to test Smart it'll be necessary to modify this condition to ensure we're not in the control of this test and not in the Smart test at all, e.g. something like:

```suggestion
	!isInVariantSynchronous(integrateCriteo, 'control') &&
	!isInABTestSynchronous(integrateSmart);
```

Reply:
You don’t have to if you manage your offsets right.

Reply:
This no longer applies!